README.md

```diff
@@ -1,22 +1,31 @@
-# PostHTML Content
+[![NPM][npm]][npm-url]
+[![Dependencies][deps]][deps-url]
+[![DevDependencies][devdeps]][devdeps-url]
```

Comment:
Rather than dev dependencies, would prefer to have a tests badge here from travis. All the badges we need were here previously if you need the urls!